src/lwc/rdActiveSchedule/rdActiveSchedule.js-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>46.0</apiVersion>
+    <description>Displays the active recurring donation schedule details.</description>
+    <isExposed>true</isExposed>
+    <masterLabel>Recurring Donation Schedules</masterLabel>
```

Comment:
@npsp-reedestockton  - Is this the label that appears in the UI when you go to add the component to a record page?

Reply:
Yes, it is.  Should probably change to Active Schedules.